### PR TITLE
Moved AudioPlayer and NftCard from UI lib to main project

### LINKED ISSUE #8286: Migration `NeoAudioPlayer` and `NeoNftCard` from UI to main project
### Is your feature request related to a problem?

NeoAudioPlayer is in libs/ui but in contains main logic

### Describe the solution you would like

NeoAudioPlayer component should be in main project

### Describe alternatives you have considered

--

### Screenshots

_No response_

### Are there opened related issues?

_No response_

### PR DESCRIPTION
**Pull request by shashkovdanil**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #8286
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [x] I've tested it at </ksm/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP&usdamount=0)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at c5b31df</samp>

Refactored and improved several components related to NFT cards and audio players. Replaced `NeoNftCard` and `NeoAudioPlayer` with `NftCard` and `AudioPlayer`, and updated their dependencies, styles, and types. Removed unused or duplicated components from the `ui` library and the `nft-gallery` repository.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at c5b31df</samp>

> _`NeoNftCard` gone_
> _Refactored for clarity_
> _Autumn of old code_

### CODE REVIEW

File: components/shared/audioPlayer/AudioPlayer.vue [hunk 1]
```diff
@@ -121,7 +121,7 @@ const pushToActionStack = ({ promise, reject }) => {
 const play = (time?: number) => {
   return new Promise((resolve, reject) => {
     if (!canPlay.value) {
-      return reject(new Error("Player: Can't play"))
+      return reject(new Error("Player: Can't play")) // eslint-disable-line quotes
```
Comment (roiLeo) [inline components/shared/audioPlayer/AudioPlayer.vue:124]: use single quote or backtick

File: components/shared/nftCard/NftCard.scss
Comment (roiLeo) [inline components/shared/nftCard/NftCard.scss:1]: I would move this file inside `assets/styles/components/NftCard.scss`

Comment (roiLeo) [reply components/shared/nftCard/NftCard.scss:1]: What's wrong? this is how it's done for some components (gallery item, collection card, top collection etc..)

Comment (roiLeo) [reply components/shared/nftCard/NftCard.scss:1]: This looks good

**Review by roiLeo — APPROVED** [review]
conflicts

**Review by daiagi — APPROVED** [review]

**Comment by daiagi** [comment]
@prury  plz go over this make sure nothing broken
code lgtm

**Comment by prury** [comment]
- cover not present: 
https://deploy-preview-8550--koda-canary.netlify.app/ksm/gallery/15297998-900d19dc7d3c444e4c-PLANETPO-DEEPOLOGIC_PLANET_PO_EP-00000004
image

- explorer 1
image

- explorer 2
image
https://deploy-preview-8550--koda-canary.netlify.app/ksm/explore/items?listed=true&page=13

- items not loading: 
https://deploy-preview-8550--koda-canary.netlify.app/ksm/gallery/17540187-c8205518d881699b3e-%F0%9F%94%8A-FROM_WORTH_KEEPING-00000028

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.64 USD/DOT ~ 3.472 $DOT
🧗 [16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP](https://polkadot.subscan.io/account/16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP)
🔗 [0x97de7d1d2396c5c98f0be846405920dbac8a5cf10fd45d19eba0f07464c3b4d3](https://polkadot.subscan.io/extrinsic/0x97de7d1d2396c5c98f0be846405920dbac8a5cf10fd45d19eba0f07464c3b4d3)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**